### DISCUSSION TITLE
Frequency of non-specific API endpoints

### DISCUSSION THREAD

**Discussion by crstauf** [upptime/upptime#336] [Q&A]
With what frequency are the non-specific API endpoints (`response-time.json`, `uptime.json`) updated? 

What value is used for the non-specific API endpoints (`response-time.json`, `uptime.json`)? The other endpoints indicate one day, one month, one week, and one year; are the non-specific endpoints for the most recent reading, or for the life of the uptime monitor?

Some documentation indicating this would be helpful.

**Comment by AnandChowdhary** [comment]
The non-specific ones are all-time, yes. You're right, we should document this somewhere.
